installer/gentoo/prepare

```diff
@@ -0,0 +1,179 @@
+#!/bin/sh -e
+# Copyright (c) 2017 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This is a distro-specific continuation of the prepare.sh script.
+
+DISTROAKA='gentoo'
+
+# Set global use flags, separated by a space
+set_flags() {
+    while [ "$#" != 0 ]; do
+         local flag=`euse -a $1 | cut -d\[ -f1`;
+	 # If don't already have the flag
+	 if [ -z "${flag}" ]; then
+	     euse -E $1 1> /dev/null
+	 fi
+	 shift
+    done
+}
+
+# Set a package specific use flag
+# Usage: set_package_flag <package> <use flag> <crouton target for comment>
+set_package_flag() {
+    # Check if the directory exists
+    if [ ! -d /etc/portage/package.use ]; then
+        mkdir /etc/portage/package.use
+    fi
+
+    # Check if the package use file exists
+    if [ ! -f /etc/portage/package.use/crouton ]; then
+        touch /etc/portage/package.use/crouton
+    fi
+
+    # Check if the requested package/use is already in the file
+    USEFLAG=`grep -e "^$1 $2$" /etc/portage/package.use/crouton | cut -f1 -d1`
+
+    # Set the use flag
+    if [ -z "${USEFLAG}" ]; then
+        echo "# Added flag $2 to $1 for crouton target $3" >> /etc/portage/package.use/crouton
+	echo "$1 $2" >> /etc/portage/package.use/crouton
+    fi
+}
+
+# Set unstable keywords for a package
+# Usage: keyword_package <package> <crouton target for comment>
+keyword_package() {
+    # Check if the directory exists
+    if [ ! -d /etc/portage/package.keywords ]; then
+        mkdir /etc/portage/package.keywords
+    fi
+
+    # Check if the package keywords file exists
+    if [ ! -f /etc/portage/package.keywords/crouton ]; then
+        touch /etc/portage/package.keywords/crouton
+    fi
+
+    # Check if the requested package is already in the file
+    UNMASK=`grep -e "^$1[[:space:]]" -e "^$1$" /etc/portage/package.keywords/crouton | cut -f1 -d1`
```

Comment:
Oh, I see, this matches how you access package.license (which I assume can be edited by other tools)